Mathlib/Data/List/Destutter.lean

```diff
@@ -212,13 +212,13 @@ theorem length_destutter' [IsEquiv α Rᶜ] (hab : ¬R a b) :
   induction l with
   | nil => simp
   | cons c cs ih =>
-    by_cases hac : (Rᶜ a c)
-    case pos =>
+    obtain hac | hac : R a c ∨ Rᶜ a c := em _
+    next =>
+      have hbc : ¬Rᶜ b c := mt (_root_.trans hab) (Classical.not_not.2 hac)
+      simp [destutter', if_pos hac, if_pos (Classical.not_not.1 hbc)]
+    next =>
       have hbc : ¬R b c := trans (symm hab) hac
```

Comment:
Why not just
```suggestion
    · have hbc : ¬Rᶜ b c := mt (_root_.trans hab) (Classical.not_not.2 hac)
      simp [destutter', if_pos hac, if_pos (Classical.not_not.1 hbc)]
    · have hbc : ¬R b c := trans (symm hab) hac
```

Reply:
Done, also dropped an extra couple `Classical.`s